maquetado/scss/styles.scss

```diff
@@ -150,3 +151,79 @@
   display: flex;
   flex-direction: row;
 }
+
+@media only screen and (max-width: 1024px) {
+
+  .body-detail {
+    display: flex;
+    align-items: center;
+    flex-direction: column;
```

Comment:
Same, you could use `flex-direction: column-reverse` here instead of setting order property in arrow redirect class. We usually use order when we have more than 2 elements and we want to set an specific order that we can't make through `flex-direction`

Reply:
In this case I think it's correct to do this with order, because if we add more stuff to the view we probably won't want to have them reversed, just the "back" button at the bottom

Reply:
@damfinkel Perfect, I see the point. Let's leave it like this then :+1: